user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_DiscoveryAndProvisioning.md

```diff
@@ -0,0 +1,160 @@
+---
+uid: IDP_Tutorial_DiscoveryAndProvisioning
+keywords: idp tutorial, idp kata
+---
+
+# Getting started with IDP and onboarding new equipment
+
+This tutorial will show you how to install IDP and onboard new equiment.
+
+Expected duration: 20 minutes.
+
+<!-- TODO: Uncomment and fill in with correct info when Kata is released.
+> [!TIP]
+> See also: [Kata #xx: IDP Introduction](https://community.dataminer.services/courses/kata-xx/)
+-->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.0-CU0 or higher that is
+
+  - [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+  - running on a Microsoft server.
```

Comment:
Microsoft server? Do you mean Windows?
And why the requirement? Isn't it always the case as DataMiner runs always on Windows?

Reply:
Is it possible at all to make it also compatible with Windows 10/11?
Some users may want to test and play around on their local development environment

Reply:
Technically yes, but that would require an update of the IDP package. This is currently tailored towards production systems.